docs/installation/ubuntulinux.md

```diff
@@ -12,7 +13,9 @@ parent = "smn_linux"
 
 Docker is supported on these Ubuntu operating systems:
 
+- Ubuntu Wily 15.04
 - Ubuntu Vivid 15.04
+- Ubuntu Utopic 14.10
```

Comment:
Yes, and it will be forever, only the last docker version in that repo is docker 1.7.1, newer versions of docker will not be published (which confused some people)